pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -821,6 +818,10 @@ self: super: {
   # Needs QuickCheck <2.10, HUnit <1.6 and base <4.10
   pointfree = doJailbreak super.pointfree;
 
+  # Test suite build depends on ancient tasty 0.11.x.
+  # Depends on ancient base <4.12
+  cryptohash-sha512 = doJailbreak super.cryptohash-sha512;
```

Comment:
A couple things here:

1. Can you rebase this on `haskell-updates` (and change base branch here as well)?  We generally send all Haskell PRs to the `haskell-updates` branch (since hackage-packages.nix gets regenerated automatically on that branch).
2. Can you remove `cryptohash-sha512` from the list of broken packages in `configuration-hackage2nix.yaml` (instead of editing hackage-packages.nix directly)?  `hackage-packages.nix` gets updated everyday automatically.
3. Can you link to the upstream issue about fixing the tasty and base version constraints here in the code?  This is so that the haskell-ecosystem maintainers in nixpkgs know when `doJailbreak` can be removed.

Reply:
> - [X] Can you rebase this on haskell-updates (and change base branch here as well)? We generally send all Haskell PRs to the haskell-updates branch (since hackage-packages.nix gets regenerated automatically on that branch).
> - [X] Can you remove cryptohash-sha512 from the list of broken packages in configuration-hackage2nix.yaml (instead of editing hackage-packages.nix directly)? hackage-packages.nix gets updated everyday automatically.
> - [X] Can you link to the upstream issue about fixing the tasty and base version constraints here in the code? This is so that the haskell-ecosystem maintainers in nixpkgs know when doJailbreak can be removed.

:-D

@cdepillabout Thanks for the explanation of the haskell+nixpkgs process :+1:

We could add a version check because the jailbreak can be removed when the package gets updated. If this one gets a new version it will certainly include the fix. Not sure how to make that in a maintainable way.